### DISCUSSION TITLE
Add ability to hide discussion threads without having to delete them

### DISCUSSION THREAD

**Discussion by DaneEveritt** [community/community#2876] [Discussions] [answered]
Howdy, currently running into something I foresee becoming tedious in our discussions, which is the inability to take moderation action on a discussion and also hide it from the normal listing without also just deleting the issue.

I'd like to be able to let people know why it isn't appropriate for our discussion setup without also leaving it there on the main listing. Right now all I can do is comment, lock, hope the person sees the feedback, and then delete it later on.

I'd like to be able to just "hide" a discussion making it still accessible with the link, but otherwise not visible in the main listing.

**Comment by becca** [comment, marked as answer]
Agreed on this being a necessary moderation feature that does not currently exist today. 

We are tracking this request and will update you as soon as we are able to prioritize landing a solution. cc// @evi-liu
